core/drand_control.go

```diff
@@ -41,10 +42,14 @@ func (d *Drand) InitDKG(c context.Context, in *control.InitDKGPacket) (*control.
 	}
 	d.idx = idx
 
+	entropyReader := entropy.NewEntropyReader(in.Entropy, in.UserOnly)
+
 	d.nextConf = &dkg.Config{
-		Suite:    key.KeyGroup.(dkg.Suite),
-		NewNodes: d.group,
-		Key:      d.priv,
+		Suite:          key.KeyGroup.(dkg.Suite),
+		NewNodes:       d.group,
+		Key:            d.priv,
+		Reader:         entropyReader,
+		UserReaderOnly: entropyReader.GetUserOnly(),
```

Comment:
I'd take the flag from the in packet instead, the reader should only be a reader, nothing more !